ui/src/app/views/workspace-edit/component.ts

```diff
@@ -24,6 +24,10 @@ export class WorkspaceEditComponent implements OnInit {
     this.workspace = {
       name: '',
       description: '',
+      /**
+       * TODO: use the free billing project created for the user once registration work is done
+       */
+      namespace: 'all-of-us-broad',
```

Comment:
What's the relation between namespace and billing project? Is it 1:1? I assume the namespace is not identical to the billing project.

Reply:
I believe that based on some of our discussions it is a 1:1 relationship, and may in fact be identical. (Because the billing project created is distinct for each user, it has a guarantee of uniqueness. I can't be 100% sure on that answer though.)

Reply:
Namespace and billing project are 1:1 in FC; each billing project has a name (also a GCP project name) which forms a namespace for workspaces.